base/build.sh

```diff
@@ -1,14 +1,17 @@
 #!/bin/bash -e
```

Comment:
Is there any reason, why you are building dedicated base builds?
I dont understand why there is this additional step, because its easier to make a full-dockerfile for each platform with seperate BASE and APPLICATION Layer, so docker caching works anyway

Reply:
We could run the apache one into a single file but the fpm builds build an apache and an alpine version.  Both depending on the base build for their distro.  We could refactor them into single files and do away with base build.

Go ahead if you want.  I'm not wedded to this strategy.

Reply:
I will think about this - maybe i have a strategy that can reduce complexity of this and we dont need the base builds anymore